README.md

```diff
@@ -1,84 +1,73 @@
+To use this module, follow the instructions to [add a module from the Viam registry](https://docs.viam.com/registry/configure/), choose the **Motor** component, and select the `odrive:serial` or `odrive:canbus` model from the [`odrive` module](https://app.viam.com/module/viam/odrive), depending on the type of motor you are using.
 
 ## Configuration
 
-To add an `canbus` or `serial` motor, update the JSON from the template with the following:
+### Prepare your single-board computer
+
+1. If you haven’t already, [install viam-server](https://docs.viam.com/installation/) on your single-board computer.
+2. Install [`odrivetool`](https://docs.odriverobotics.com/v/latest/interfaces/odrivetool.html), [`python-can`](https://pypi.org/project/python-can/), [`cantools`](https://pypi.org/project/cantools/), and the [Viam Python SDK](https://python.viam.dev).
+3. Enable SPI communication on your single-board computer to support the use of several common CANHats. If you are using a Raspberry Pi, see [these instructions](https://docs.viam.com/installation/prepare/rpi-setup/#enable-communication-protocols). Other single-board computers may have other ways of setting up CANBus communications; consult the documentation for your specific board for further guidance.
+
+### Configure your ODrive motor hardware
+
+> [!NOTE]  
+> When making the initial connection to set up the ODrive, you must make a `serial` connection. If you intend to use a `canbus` connection, you can either leave the serial connection plugged in, or remove it and just leave the CANH and CANL pins wired after you initially set up the ODrive.
+
+Use `odrivetool` to configure and tune your motor properly. See the [ODrive documentation](https://docs.odriverobotics.com/v/latest/getting-started.html) for more information. This configuration remains on the same ODrive motor controller across reboots, and only changes if you run `odrivetool` again to reconfigure your ODrive motor.
+   * Note that `iq_msg_rate_ms` in the config defaults to `0`, and you **must** set this to around `100` to use the [motor API's `SetPower` method](https://docs.viam.com/components/motor/#setpower).
+   * See the section [Add an `odrive_config_file`](#add-an-odrive_config_file) for more information on dynamic configuration.
+* See the [ODrive CAN documentation](https://docs.odriverobotics.com/v/latest/can-guide.html) for detailed information on how to set up CAN on your ODrive.
+
+### Connect your ODrive to your single-board computer
+
+Connect your ODrive motor to your single-board computer in one of the following ways:
+
+* For a `serial` connection: plug the [USB Isolator for ODrive](https://odriverobotics.com/shop/usb-c-to-usb-a-cable-and-usb-isolator) into a USB port on your board, and then plug a USB-C to USB-A cable from the isolator to the ODrive.
```

Comment:
maybe switch USB-C and USB-A in this sentence so it's in the same order as isolator and Odrive. Meaning USB-A goes to the isolator and USB-C goes to the ODrive